src/components/TokenInput.tsx

```diff
@@ -24,25 +24,29 @@ import { useTranslation } from "react-i18next"
 
 interface Props {
   symbol: string
+  name?: string
   max?: string
   inputValue: string
   onChange: (value: string) => void
   disabled?: boolean
   error?: boolean
+  allowDecimalOverflow?: boolean
   helperText?: string
 }
 
 function TokenInput({
   symbol,
+  name,
   max,
   inputValue,
   onChange,
   disabled,
   error,
+  allowDecimalOverflow,
```

Comment:
what is decimal overflow in this context?

Reply:
I was going to expand the use case of `TokenInput`.
Currently, it has limited with `isValidPrecision` and this value is from `TOKEN_MAP[symbol]`, more precisely the `pool_names`.
So, to use this component for the tokens outside of pool like, SDL, and veSDL without changing much codes, I added this props.

But we can implement this logic if necessary.

Reply:
I removed this variable, and instead of this variable, I added `decimalLength` props, so user can give the decimal length even though there is no token on POOL_MAP